### ISSUE TITLE
Desync window not displayed, corrupts UI

### ISSUE THREAD

**Issue by Sheeo** [FAForever/fa#536]

**Comment by Crotalus** [comment]
Error:
~~~
info: Hooked /lua/ui/dialogs/desync.lua with /mods/em/hook/lua/ui/dialogs/desync.lua
warning: Error running '/lua/ui/uimain.lua:ShowDesyncDialog': ...faforever\gamedata\lua.nx2\lua\ui\dialogs\desync.lua(69): access to nonexistent global variable "CreateBorder"
~~~

**Comment by ChrisKitching** [comment]
Should be fixed in 0c8ddd0af6d0d45887b0c54f4fc2c485abe0224a. Will close once that gets published.
